tempest/api/workloadmgr/workload_reassign/test_workload_reassign.py

```diff
@@ -284,3 +284,210 @@ def test_03_workload_reassign(self):
 
 
 
+
+    '''
+    OS-2061 - workload reassign with temp user and same project after deletion of temp user.
+    http://192.168.15.51/testlink/linkto.php?tprojectPrefix=OS&item=testcase&id=OS-2061
+    '''
+    @decorators.attr(type='workloadmgr_api')
+    def test_04_workload_reassign(self):
+        try:
+
+            reporting.add_test_script(str(__name__) + "_same_user_same_tenant_after_deletion_of_user")
+
+            #create a temp user...
+            self.temp_user_id = self.createUser()
+
+            #assign trustee role
+            role_id = self.get_role_id(tvaultconf.trustee_role)
+            if len(role_id) != 1:
+                raise Exception("Role ID not returned")
+
+            #assign role to user and current project.
+            if self.assign_role_to_user_project(CONF.identity.tenant_id,
+                    self.temp_user_id, role_id[0], False):
+                LOG.debug("Role assigned to user and project")
+                reporting.add_test_step("Role assignment to user and project", tvaultconf.PASS)
+            else:
+                raise Exception("Role assignment to user and project failed")
+
+            ### Create vm and workload ###
+            self.created = False
+            vm_id = self.create_vm(vm_cleanup=True)
```

Comment:
default value of vm_cleanup is True so it can be removed

Reply:
Made required minor changes.